### DISCUSSION TITLE
Behavior of Flashlight during Break Time is not working as intended in lazer (compared to stable)

### DISCUSSION THREAD

**Discussion by Nekomosh004** [ppy/osu#20667] [Ideas] [answered]
Whenever there is a break time in a beatmap, lazer removes the dark area of Flashlight on the entire playfield. I noticed this happens after 2 seconds of a break time.

https://user-images.githubusercontent.com/82471957/194765189-c82ef758-ec70-441c-9b0c-f4dacd419ed6.mp4

https://user-images.githubusercontent.com/82471957/194765138-ecb756fb-d079-42a6-83b4-bbae771ac6e2.mp4

This also happens on the other rulesets as well.
Video recordings using osu! classic (2013) skin for the other rulesets:

| stable | lazer |
| --- | --- |
| [taiko](https://user-images.githubusercontent.com/82471957/194765376-4ba0b380-e333-4191-af98-a367ab6bd3aa.mp4) | [taiko](https://user-images.githubusercontent.com/82471957/194765372-19ab9b52-0e5f-4d9f-8827-04d36cefe6c2.mp4) |
| [catch](https://user-images.githubusercontent.com/82471957/194765334-75899a15-1745-4a69-b7db-22ecfd5f1705.mp4) | [catch](https://user-images.githubusercontent.com/82471957/194765243-7dd65cff-6c22-44ac-9432-d99a3a590c98.mp4) |
| [mania](https://user-images.githubusercontent.com/82471957/194765391-cf80d216-7585-430c-9b66-d2ea7c88dcce.mp4) | [mania](https://user-images.githubusercontent.com/82471957/194765389-76ff9f48-1bf3-44bc-9027-985d9c5976a8.mp4) |

These are the dim settings I have when recording these videos:
![image](https://user-images.githubusercontent.com/82471957/194765987-c15d6c7f-4da0-4ed8-8eb6-53e59b752d77.png)
![image](https://user-images.githubusercontent.com/82471957/194766032-3d2f99fb-d254-4478-9e3b-77f4994483a6.png)

I hope this is something you consider to fix as well. Thank you!

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/pull/20714
